main.js

```diff
@@ -0,0 +1,51 @@
+const BASE_URL = "https://opentdb.com/api.php?amount=10";
+
+const form = document.querySelector('form');
+form.addEventListener('submit', submitEvent => {
+    submitEvent.preventDefault();
+    triviaQuestions(BASE_URL)
+})
+
+function triviaQuestions(url){
+    fetch(url)
+        .then((response) => response.json())
+        .then(({results}) => { // results is destructuring object
+            console.log(results) // an array of cards
+            results.forEach(element => { createCard(element)    
+            });
+        })
+        .catch((error) => {
+            console.log(error)
+        })
+}
+
+function colorForDifficulty(cardDifficulty){
+    return cardDifficulty === "hard" ? "ee0877" : cardDifficulty === "medium" ? "#fdffab" : "#0ced26";
+}
```

Comment:
Great idea to have a function to do this, but I'd stay away from nested ternaries—they harm readabilty (you'll find many articles online warning the same!). You _could_ use an `if`/`else if` chain here, but an even better approach would be to make a `colorForDifficulty` _object_ where each key is the difficulty name and the value is the color to use. That way, you could say `article.style.borderColor = colorForDifficulty[difficulty]` and get the right color!